essential-projects/sequelize_connection_manager
# Feature/implement persistence layer

## What did you change?

Implement a binding factory that creates and manages SQL connections made with sequelize.

The factory runs as a static function and can be used from anywhere. 

Created connections are stored in an object, associated with a hash.
When a requesting module wants to connect to a database for which there is a connection already active, the active connection is returned. 

## How can others test the changes?

Use the factory to create database connections. The configs passed to the factory must be compatible with sequelize.
See:
http://docs.sequelizejs.com/manual/installation/getting-started.html#setting-up-a-connection

## PR-Checklist

Please check the boxes in this list after submitting your PR:
